### ISSUE TITLE
Seren fails when trakt is not working 

### ISSUE THREAD

**Issue by PGB9999** [nixgates/plugin.video.seren#854]
Trakt has been down for the last few days, and Seren won't start.

Is there a way to disable trakt support in seren until trakt is back up?

I have revoked authorisation in the seren settings, but it seems that the integration is much tighter than it used to be.

**Comment by drinfernoo** [comment]
No. We're aware that Trakt has been having issues, but Seren is (and always has been) a Trakt-first add-on. Be patient, they'll figure it out soon.

**Comment by PGB9999** [comment]
I realise Trakt is an integral part of Seren, but it woud have been good (in this situation) to be aboe to bypass trakt, accepting that tracking of played items etc would hve to be updated after the fix.

**Comment by nixgates** [comment]
The issue with adding another tracker like Simkl, is the whole addon is written with Trakt in mind. We pass trakt Ids around to play, open, etc. We store all metadata based on Trakt Ids. Almost everything related to show/movie items are all based around Trakt.

Its incredibly tightly woven and would be a massive undertaking to re-work at this point. Not something I think is feasible and I say that with a heavy heart. The project is a pet love of mine and it was sad to see it so crippled.

With that in mind, the recent issue with Trakt is an unprecedented issue and not one that i believe has been seen befofe for a metadata provider that Im aware of. At least in the several years I've been around. 

Usually the worst we see is about half a day and maybe once or twice a year on average for Trakt. Their service has gotten incredibly stable over the last few years.

If this becomes a more frequent issue I will revisit this but I think for the moment its best we let the ball roll for a little while instead of moving to an almost complete re-write of how handle all our local data.

**Comment by npmstart-pray** [comment]
Maybe you should take a look at how Simkl imports trakt histories…it might solve some time, or make the mountain ahead more of a molehill. If you’re going to pursue it, that is. Merry Christmas, Doc, and thanks for all you do!Sent from nowhere near my ready room with my new tricorderOn Dec 23, 2022, at 20:11, nixgates ***@***.***> wrote:﻿
The issue with adding another tracker like Simkl, is the whole addon is written with Trakt in mind. We pass trakt Ids around to play, open, etc. We store all metadata based on Trakt Ids. Almost everything related to show/movie items are all based around Trakt.
Its incredibly tightly woven and would be a massive undertaking to re-work at this point. Not something I think is feasible and I say that with a heavy heart. The project is a pet love of mine and it was sad to see it so crippled.
With that in mind, the recent issue with Trakt is an unprecedented issue and not one that i believe has been seen befofe for a metadata provider that Im aware of. At least in the several years I've been around.
Usually the worst we see is about half a day and maybe once or twice a year on average for Trakt. Their service has gotten incredibly stable over the last few years.
If this becomes a more frequent issue I will revisit this but I think for the moment its best we let the ball roll for a little while instead of moving to an almost complete re-write of how handle all our local data.

**Comment by drinfernoo** [comment]
Other issues advocating for Simkl support: #480, #670, #764

**Comment by npmstart-pray** [comment]
So maybe it might be worth considering?  Especially for the folk who are looking for an anime tracker?  And those of us who want some backup if/when trakt goes down again?Then again, there’s been a request for kodi to start using xml to speed searches up overall for about a decade now, so either way, there seems to be a desire for change among kodi users and not just addons like Seren. Idk how it might moot these requests, but different is sometimes the thing to try…Sent from nowhere near my ready room with my new tricorderOn Dec 25, 2022, at 22:47, drinfernoo ***@***.***> wrote:﻿
Other issues advocating for Simkl support: #480, #670, #764
